packages/components/src/components/as-category-widget/as-category-widget.scss

```diff
@@ -0,0 +1,81 @@
+@import '../../../../styles/src/core/_dev';
+
+$bar--height: get-spacing(1);
+
+as-category-widget {
+  display: block;
+
+  @include padding(4);
+
+  :root {
+    --description--color: $color-type-20;
+    --category-bar--background-color: $color-ui-20;
+    --bar--height: $bar--height;
+  }
+
+  .as-category-widget__description,
```

Comment:
we use `&__description` in the styles, I think we should pick one and be consistent all over the project

Reply:
Yes, I think so but the thing here is that it is not `.as-category-widget` but `as-category-widget`. So whether we wrap these styles in `.as-category-widget` or we set that class to the host element and transform `as-category-widget` to `.as-category-widget`.

What do you prefer?

Reply:
oooouch, you are right... so its fine this way